pep-0710.rst

```diff
@@ -171,16 +171,146 @@ from the installer's cache.
 Backwards Compatibility
 =======================
 
-Since this PEP specifies a new file in the ``.dist-info`` directory, there are
-no backwards compatibility implications to consider in the ``provenance_url.json``
-file itself. Also, this proposal does not make any changes to the
-``direct_url.json`` described in :pep:`610` and
-:ref:`its corresponding canonical PyPA spec <direct-url>`.
+Following the :ref:`packaging:recording-installed-packages` specification,
+installers may keep additional installer-specific files in the ``.dist-info``
+directory.  To make sure this PEP does not cause any backwards compatibility
+issues, there was conducted a research whether introducing the
+``provenance_url.json`` file in the ``.dist-info`` directory is feasible and
+does not cause any backwards compatibility issues in the Python packaging
+ecosystem.
+
+A specification for :ref:`binary-distribution-format` lists files that can be
+present in the ``.dist-info`` directory. None of these file names collide with
+the proposed ``provenance_url.json`` file from this PEP.
+
+Presence of provenance_url.json in installers and libraries
+-----------------------------------------------------------
+
+A research conducted on the existing installers, libraries, and dependency
+managers in the Python ecosystem shown below proved there are no backwards
+compatibility issues and introducing the ``provenance_url.json`` file will not
+clash with any existing tool at the time of writing this PEP.
+
```

Comment:
To note, as written this essentially duplicates the information two paragraphs above if this remains here, but if moved to a separate appendix section as I recommend, it would make sense to keep this with the revisions that follow.

```suggestion
A comprehensive survey of the existing installers, libraries, and dependency
managers in the Python ecosystem analyzed the implications of adding support for
``provenance_url.json`` to each tool.
In summary, no major backwards compatibility issues, conflicts or feasibility blockers
were found as of the time of writing of this PEP.
```

* Revise following the form above and to be more appropriate for an independent appendix
* Fix grammar errors
* Avoid excessive verbosity
* Use more appropriate and qualified wording